doc/plugins/eu.esdihumboldt.hale.doc.user/toc.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<toc label="hale studio User Guide">
+<toc label="hale»studio User Guide">
```

Comment:
Wrong or just not correctly displayed here?

Reply:
Interesting - looks different in "Conversation" than in "Files changed"

Reply:
I checked the encoding and it looks good to me in UTF-8. Not sure why it displays wrong on the "Files changed" tab.